lib/web_ui/lib/src/engine/html/path/path.dart

```diff
@@ -43,18 +64,14 @@ class SurfacePath implements ui.Path {
   ui.PathFillType _fillType = ui.PathFillType.nonZero;
   // Skia supports inverse winding as part of path fill type.
   // For Flutter inverse is always false.
-  bool _isInverseFillType = false;
+  final bool _isInverseFillType = false;
```

Comment:
A final bool field seems like an unnecessary variable.

Reply:
Removed and updated code that was using it.